seaborn/utils.py

```diff
@@ -219,9 +219,10 @@ def despine(fig=None, ax=None, top=True, right=True, left=False,
     """Remove the top and right spines from plot(s).
 
     fig : matplotlib figure, optional
-        Figure to despine all axes of, default uses current figure.
+        Figure to despine all axes of, defaults to current figure.
     ax : matplotlib axes, optional
-        Specific axes object to despine.
+        Specific axes object to despine when a figure is not
```

Comment:
I think this would be clearer as 

> Specific axes object to despine. Ignored if ``fig`` is provided.

Reply:
Thank you for the suggestion, I updated the wording accordingly.